src/components/graph/search/GraphSearch.tsx

```diff
@@ -252,39 +240,25 @@ function GraphSearch(props: Props) {
 							setIsIdSearchDisabled={setIsIdSearchDisabled}
 							closeModal={closeModal}
 							submittedId={submittedId}
+							submittedTimestamp={timestamp}
 							isIdMode={showModal && mode === 'history'}
 						/>
 					)}
-					{mode === 'confirm' && refreshPanels && (
-						<GraphSearchConfirm
-							onConfirm={() => {
-								if (timestamp && refreshPanels) {
-									refreshPanels(timestamp);
-								}
-								setShowModal(false);
-								setMode('timestamp');
-							}}
-							onDecline={() => setShowModal(false)}
-						/>
-					)}
 					<div className='graph-search__switchers'>
-						{mode === 'confirm' ? null : (
-							<>
-								<button className={dateButtonClassName} onClick={() => onModeSelect('timestamp')}>
-									Timestamp
-								</button>
-								<button className={idButtonClassName} onClick={() => onModeSelect('history')}>
-									History
-								</button>
-							</>
-						)}
+						<>
```

Comment:
since the condition has been removed, now you can delete this React.Fragment (`<></>`)